flocker/node/agents/test/blockdevicefactory.py

```diff
@@ -209,10 +213,24 @@ def _aws(cluster_id, config):
         ec2_client=get_ec2_client_for_test(config),
     )
 
+
+def _gce(cluster_id, config):
+    """
+    Create an IBlockDeviceAPI provider configured to use the GCE PD
+    region where the server that is running this code is running.
+
+    :param cluster_id: The flocker cluster id.
+    :param config: Any additional configuration (possibly provider-specific)
+        necessary to authenticate a PD session.
+    :return: A PDBlockDeviceAPI instance.
+    """
+    return PDBlockDeviceAPI(**config)
+
 # Map provider labels to IBlockDeviceAPI factory.
 _BLOCKDEVICE_TYPES = {
     Providers.OPENSTACK: _openstack,
     Providers.AWS: _aws,
+    Providers.GCE: _gce,
```

Comment:
Release planning question. I don't intend on putting this in release notes until GCE support is fully tested (in our Jenkins, and all the TODOs taken care of).

However (I'll have to double check this) but I think you can technically attempt to use GCE as soon as it is in the code. Should we be putting it behind some sort of release flag (or other technical barrier) until it is "ready"?